server/app/services/applicant/ApplicantService.java

```diff
@@ -398,13 +441,39 @@ CompletionStage<Application> submitApplication(
               String programName = application.getProgram().getProgramDefinition().adminName();
               notifyProgramAdmins(applicantId, programId, application.id, programName);
 
+              CompletableFuture<Void> maybeNotifyTiSubmitterFuture;
```

Comment:
**suggestion:** Maybe you could do something like this here and below to clean it up a bit:

```
CompletableFuture<Void> maybeNotifyTiSubmitterFuture = tiSubmitterEmail.isPresent() ? maybeNotifyTiSubmitter(
                            tiSubmitterEmail.get(), applicantId, application.id, programName)
                        .toCompletableFuture() : CompletableFuture.completedFuture(null);
```